evaluate/metrics/assessment.go

```diff
@@ -0,0 +1,40 @@
+package metrics
+
+// AssessmentKey is a description for a numerical assessment value.
+type AssessmentKey string
+
+// allAssessmentKeys holds all assessment keys.
+var allAssessmentKeys []AssessmentKey
+
+func registerAssessmentKey(key string) AssessmentKey {
+	assessment := AssessmentKey(key)
+	allAssessmentKeys = append(allAssessmentKeys, assessment)
+
+	return assessment
+}
+
+var (
+	// AssessmentKeyNoExplanations means that a model did not produce additional explanations.
+	AssessmentKeyNoExplanations = registerAssessmentKey("no-explanations")
+)
+
+// Assessments holds numerical assessment metrics.
+type Assessments map[AssessmentKey]uint
+
+// Add sums two assessment objects.
+func (a Assessments) Add(o Assessments) Assessments {
```

Comment:
I think this should be "Merge" or "Union". "Add" seems to me adding to the same object, but we are adding to a new one.

Reply:
If we reuse the existing object, I would call the operation "update" then... because "merge" and "union" are more like operations that have a result.

Reply:
If we always just update the first object, then we cannot handle the case anymore where it is `nil`... So we must ensure everywhere that it is initialized.

Reply:
Yeah. That is what i mean

- Merge combines objects a and b, to return a new c
- Add adds object b to a and does not return